README.md

```diff
@@ -212,8 +212,9 @@ time a `viper.Get` request is made. It will apply the following rules. It will
 check for a environment variable with a name matching the key uppercased and
 prefixed with the `EnvPrefix` if set.
 
-`SetEnvKeyReplacer` allows you to use a `strings.Replacer` object to rewrite Env
-keys to an extent. This is useful if you want to use `-` or something in your
+`SetEnvKeyReplacer` allows you to use a `replacer` interface to rewrite Env
```

Comment:
I would place the interface definition in the readme somewhere, because it's not exported.

Reply:
I would also remove "interface" from after `replacer`: it's redundant.